organizations/v0/tests/test_views.py

```diff
@@ -0,0 +1,67 @@
+"""
+Organizations Views Test Cases.
+"""
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+
+from organizations.tests.factories import UserFactory, OrganizationFactory
+
+
+DRF_DATE_FORMAT = u'%Y-%m-%dT%H:%M:%S.%f'
+
+
+# pylint: disable=no-member, no-self-use
+class TestOrganizationsView(TestCase):
+    """ Test Organizations View."""
+    def setUp(self):
+        super(TestOrganizationsView, self).setUp()
+
+        self.user_password = 'test'
+        self.user = UserFactory(password=self.user_password)
+        self.organization = OrganizationFactory.create()
+        self.organization_url = reverse('v0:organization-list')
+
+    def serialize_organization(self, organization):
```

Comment:
As was done for the credentials API, test `OrganizationSerializer` in `test_serializers.py`. Once the serializer is validated, you can use it in your tests: `OrganizationSerializer(organization).data()`